qadence2_platforms/backends/fresnel1/device_settings.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from dataclasses import replace
+import warnings
+
+import numpy as np
+from pulser.channels import DMM
+from pulser.devices import AnalogDevice
+
+from qadence2_platforms.backends._base_analog.device_settings import DeviceSettings
+
+warnings.filterwarnings("ignore", category=UserWarning)
+warnings.formatwarning = lambda msg, *args, **kwargs: f"WARNING: {msg}\n"
+
+
+class Fresnel1DeviceSettings(DeviceSettings):
```

Comment:
Why not turning these setting classes into `dataclass` ?

Reply:
I tried, but I thought it was better to define a class to have a few methods for checking which seemed to carry too much things (`dataclass` whole hierarchy) into the table.